app/src/main/java/com/example/antonyng/level_up/MainActivity.java

```diff
@@ -2,12 +2,59 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+
+import java.util.ArrayList;
 
 public class MainActivity extends AppCompatActivity {
 
+    private static final String TAG = "MainActivity";
+
+    //vars
+    private ArrayList<String> mNames = new ArrayList<>();
+    private ArrayList<String> mImageUrls = new ArrayList<>();
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        Log.d(TAG, "onCreate: Started");
+
+        initImageBitmaps();
+    }
+
+    private void initImageBitmaps() {
+        Log.d(TAG, "initImageBitmaps: preparing  bitmaps");
+
+        mImageUrls.add("https://avatars2.githubusercontent.com/u/15978054?s=460&v=4");
```

Comment:
I believe there is a more efficient way to add multiple elements to an Array list but this approach works well. However do look into another alternative to for your personal research.

Reply:
okay